packages/harfbuzz.rb

```diff
@@ -1,26 +1,25 @@
 require 'package'
+require_relative 'freetype'
```

Comment:
Use `Package.load_package` to prevent class leak? (see #7283 for more info)
```ruby
# INSIDE the class
@freetype = Package.load_package('freetype.rb')
# @freetype.version, @freetype.git_hashtag...
```

Reply:
Done!

Reply:
ok it turns out that if you're in another directory at all from `/usr/local/lib/crew/packages` and you have a line like `@freetype = Package.load_package('freetype.rb')` in your package, `crew` totally breaks.

For instance `crew deps -b -d "harfbuzz"` absolutely no longer works.

This breaks my build setup. :) so back to `require_relative`...